migrations/sqlite/000002_add_chats.up.db.sql

```diff
@@ -0,0 +1,21 @@
+CREATE TABLE IF NOT EXISTS chats (
+id VARCHAR PRIMARY KEY ON CONFLICT REPLACE,
+name VARCHAR NOT NULL,
+color VARCHAR NOT NULL DEFAULT '#a187d5',
+type INT NOT NULL,
+active BOOLEAN NOT NULL DEFAULT TRUE,
+timestamp INT NOT NULL,
+updated_at INT NOT NULL,
+deleted_at_clock_value INT NOT NULL DEFAULT 0,
+public_key BLOB,
+unviewed_message_count INT NOT NULL DEFAULT 0,
+last_clock_value INT NOT NULL DEFAULT 0,
+last_message_content_type VARCHAR,
+last_message_content VARCHAR,
+members BLOB,
+membership_updates BLOB
+);
+
+
+DROP TABLE membership_updates;
```

Comment:
Leftover?

Reply:
we created them in migration v1, but not needed anymore, so I thought we would just drop them (not much of an issue as data migrations are not needed anymore)